scripts/network_bandwidth.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+INTERVAL="1"  # update interval in seconds
+
+network_name=$(tmux show-option -gqv "@dracula-network-bandwidth")
+
+download_bytes() {
+  case $(uname -s) in
+    Linux)
+      cat "/sys/class/net/$network_name/statistics/rx_bytes"
+      ;;
+    Darwin)
+      netstat -I "$network_name" -b | tail -n 1 | awk '{print $7}'
+      ;;
+    CYGWIN*|MINGW32*|MSYS*|MINGW*)
+      # TODO - windows compatibility
+      ;;
+  esac
+}
+
+upload_bytes() {
+  case $(uname -s) in
+    Linux)
+      cat "/sys/class/net/$network_name/statistics/rx_bytes"
+      ;;
+    Darwin)
+      netstat -I "$network_name" -b | tail -n 1 | awk '{print $10}'
```

Comment:
Would it make sense to provide the `-n` flag as well? 

I have issues currently locally with netstat hanging if I skip it, which is because (I believe, correct me if I'm wrong) it's trying to resolve addresses but we don't care about that in this case. For reference, from the man page:

```
-n    Show network addresses as numbers (normally netstat interprets addresses and attempts to 
      display them symbolically).  This option may be used with any of the display formats.
```

but having the line as `netstat -nbI "$network_name" | tail -n1 | awk '{print $10}'` works fine for me